sdk/servicebus/microsoft-azure-servicebus/pom.xml

```diff
@@ -6,10 +6,10 @@
   <modelVersion>4.0.0</modelVersion>
   <groupId>com.microsoft.azure</groupId>
   <artifactId>azure-servicebus</artifactId>
-  <version>3.7.0-beta.1</version> <!-- {x-version-update;com.microsoft.azure:azure-servicebus;current} -->
+  <version>3.6.3</version> <!-- {x-version-update;com.microsoft.azure:azure-servicebus;current} -->
 
   <name>Microsoft Azure SDK for Service Bus</name>
-  <description>Java library for Azure Service Bus</description>
+  <description>Please note, a newer package com.azure:azure-messaging-servicebus for Azure Service Bus is available as of December 2020. While this package will continue to receive critical bug fixes, we strongly encourage you to upgrade. Read the migration guide at https://aka.ms/azsdk/java/migrate/sb for more details.</description>
```

Comment:
Keep "Java library for Azure Service Bus", also add the note.

Reply:
Fair point

```suggestion
  <description>Java library for Azure Service Bus. Please note, a newer package com.azure:azure-messaging-servicebus for Azure Service Bus is available as of December 2020. While this package will continue to receive critical bug fixes, we strongly encourage you to upgrade. Read the migration guide at https://aka.ms/azsdk/java/migrate/sb for more details.</description>
```